.github/workflows/ci.yml

```diff
@@ -64,6 +64,55 @@ jobs:
       - run: yarn build
         continue-on-error: true
 
+  backend-lint:
+    name: Backend (linting)
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # Current size as of 2019-12-23: ~77 MB
+      - name: Cache cargo registry cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.cargo/registry/cache
+          key: ${{ runner.os }}-cargo-registry-cache-lint-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-cargo-registry-cache-lint-
+            ${{ runner.os }}-cargo-registry-cache-
+
+      # Current size as of 2019-12-23: ~38 MB
+      - name: Cache cargo registry index
+        uses: actions/cache@v2
+        with:
+          path: ~/.cargo/registry/index
+          key: ${{ runner.os }}-cargo-registry-index-lint-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-cargo-registry-index-lint-
+            ${{ runner.os }}-cargo-registry-index-
+
+      # Current size as of 2019-12-23: ~4 MB
+      - name: Cache cargo git db
+        uses: actions/cache@v2
+        with:
+          path: ~/.cargo/git/db
+          key: ${{ runner.os }}-cargo-git-db-lint-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-cargo-git-db-lint-
+            ${{ runner.os }}-cargo-git-db-
+
+      - name: Install Rust
+        run: |
+          rustup set profile minimal
+          rustup update 1.49.0
+          rustup default 1.49.0
```

Comment:
if we don't pin it, we risk CI failing once new stable Rust versions are released, and I guess it would make sense for us to also run the tests on the same version that we're running in production, right? (have to admit that I don't know what exactly we're running in prod though)

Reply:
Our production versions is selected by the `RustConfig` file. There is usually a window of a few days between a new release (which is picked up automatically on CI) and bumping the compiler in production. I don't have any concerns with this small lag, as it is extremely unlikely we would land something that works in `stable` but is broken in `stable - 1` during this window.

For `fmt` and `clippy`, we've been been tracking stable for a few years without issue. Plus, I usually run on nightly locally and patch any new clippy findings well before they land on stable.